doc/building.dox

```diff
@@ -58,42 +58,45 @@ Various examples are provided by the @ref building-examples "Example repository"
 @subsection building-packages-vcpkg Vcpkg packages on Windows
 
 Magnum is available as a [Vcpkg](https://github.com/Microsoft/vcpkg) package.
-After setting up Vcpkg like described in the README, you can install latest
-stable version of Magnum with all its dependencies like this:
+After setting up Vcpkg as described in their
+[README](https://github.com/Microsoft/vcpkg#quick-start), it is recommended to
+set the "default triplet". You can do this by setting the `VCPKG_DEFAULT_TRIPLET`
```

Comment:
The default settings always works with vcpkg commands. It doesn't work with the toolchain, but is not intended to.

I find the default triplet to be crucial and is not explicitly mentioned in their README Quick Start. Hence I believe this will prevent a lot of irritation for a lot of users.